contents/docs/integrate/proxy.mdx

```diff
@@ -5,12 +5,15 @@ showTitle: true
 ---
 
 A reverse proxy allows you to send events to PostHog Cloud using your own domain.
-This can have a number advantages, most notably being that events sent to your own domain won't be intercepted by tracker blockers.
-By sending events to your own domain, you'll be able to track more users and more events, without having to host PostHog yourself.
 
-> Note: PostHog Cloud requires that the proxy sets the `Host` header to `eu.posthog.com` for requests sent to https://eu.posthog.com
-> and `app.posthog.com` for requests sent to https://app.posthog.com. The instructions below do not account for this. PR's with the requested
-> changes welcome 🙏
+This means that events are sent from your own domain and are less likely to be intercepted by tracking blockers. You be able to capture more usage data without having to self-host PostHog.
+
+Setting up a reverse proxy means setting up a service to redirect requests from a subdomain you choose (like `e.yourdomain.com`) to PostHog. It is best practice to use a subdomain that does not include `posthog`, `analytics`, `tracking`, or other similar words. 
+
+You then use this subdomain as your instance host in the initialization of PostHog instead of `app.posthog.com` or `eu.posthog.com`.
+
+> **Note:** PostHog Cloud requires that the proxy sets the `Host` header to `eu.posthog.com` for requests sent to `https://eu.posthog.com`
+> and `app.posthog.com` for requests sent to `https://app.posthog.com`.
 
 ## Deploying a reverse proxy
```

Comment:
It might be good to explicitly call out the fact that we expect the `Host` header to be set correctly (eg app.posthog.com or eu.posthog.com)